src/main/java/xbot/common/controls/actuators/wpi_adapters/CANSparkMaxWpiAdapter.java

```diff
@@ -247,6 +247,11 @@ public Voltage getVoltage() {
                 this.internalSparkMax.getAppliedOutput() * internalSparkMax.getBusVoltage());
     }
 
+    @Override
+    public void setVoltageRange(Voltage minVoltage, Voltage maxVoltage) {
+        log.warn("setVoltageRange: Voltage range is not supported by SparkMax");
```

Comment:
⭐ how would code that wants to be device agnostic handle this, should there maybe be a call to check for what capabilities a motor controller supports so they only try to set what it supports?

Reply:
Won't fix in this PR, but I'll think about it.